src/kubernetes_api/swagger.clj

```diff
@@ -149,11 +149,42 @@
                                            [[verb operation]])))
                                (into {}))))))
 
+(defn default-paths [paths]
+  {"/apis/" (get-in paths ["/apis/"])
+   "/api/" (get-in paths ["/api/"])})
+
+(defn filter-api
+  "Returns the paths that starts with the given api and version
+   e.g. /apis/{api}/..."
+  [api paths]
+  (into {}
+        (filter #(string/starts-with? (first %) (str "/apis/" api)) paths)))
+
+(defn from-apis
+  "Returns the default paths and the paths for the given apis"
+  [apis paths]
+  (let [api-paths (mapv #(filter-api % paths) apis)]
+    (into (default-paths paths)
+          api-paths)))
+
+(defn filter-paths
+  "Returns an updated schema with the paths for the given apis"
+  [schema apis]
+  (update schema :paths (partial from-apis apis)))
+
+(defn filter-api-paths
+  "Retrieves the schema from the given apis"
+  [swagger {{:keys [apis]} :openapi}]
+  (if (seq apis)
+    (filter-paths swagger apis)
+    swagger))
```

Comment:
I think we should always filter paths, using a set of default API groups. This way we can make the client more consistent. WDYT?